pkg/gofr/migration/cassandra_test.go

```diff
@@ -0,0 +1,110 @@
+package migration
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+
+	"gofr.dev/pkg/gofr/container"
+	"gofr.dev/pkg/gofr/testutil"
+)
+
+func cassandraSetup(t *testing.T) (migrator, *container.MockCassandra, *container.Container) {
+	t.Helper()
+
+	mockContainer, mocks := container.NewMockContainer(t)
+
+	mockCassandra := mocks.Cassandra
+
+	ds := Datasource{Cassandra: mockContainer.Cassandra}
+
+	ch := cassandraDS{Cassandra: mockCassandra}
+	mg := ch.apply(&ds)
+
+	mockContainer.Cassandra = mockCassandra
+
+	return mg, mockCassandra, mockContainer
+}
+
+func Test_CassandraCheckAndCreateMigrationTable(t *testing.T) {
+	mg, mockCassandra, mockContainer := cassandraSetup(t)
```

Comment:
What does `mg` stands for ? Is it an acronym for **migrator** ? If yes we can just use the name `migrator`